app/models/spree_avatax/config.rb

```diff
@@ -11,6 +13,17 @@ class << self
       attr_accessor :sales_invoice_generate_error_handler
       attr_accessor :sales_invoice_commit_error_handler
       attr_accessor :sales_invoice_cancel_error_handler
+
+      def timeout
+        (config = last) ? config.timeout : DEFAULT_TIMEOUT
```

Comment:
Sure, I'm fine with adding a `most_recent` helper method or something like that.